pkgs/development/python-modules/navec/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, numpy
+, razdel
+, gensim
+, pytestCheckHook
+}: buildPythonPackage rec {
+  pname = "navec";
+  version = "0.10.0";
+  format = "setuptools";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-TyNHSxwnmvbGBfhOeHPofEfKWLDFOKP50w2QxgnJ/SE=";
+  };
+
+  propagatedBuildInputs = [ numpy razdel ];
+  nativeCheckInputs = [ pytestCheckHook gensim ];
+  disabledTests = [ "test_gensim" ];
```

Comment:
Could you add a comment about why we disabled it?

Reply:
I've added little comment into `navec` commit. Thanks for mentioning it!